master/internal/api_user.go

```diff
@@ -30,7 +31,9 @@ const determinedName = "determined"
 var (
 	errExternalSessions = status.Error(codes.PermissionDenied, "not enabled with external sessions")
 	errUserNotFound     = status.Error(codes.NotFound, "user not found")
-	latinText           = regexp.MustCompile("[^[:graph:]\\s]")
+	errUserNotFoundRBAC = status.Error(codes.NotFound,
+		"user not found, please check your permissions.")
```

Comment:
can this use the util you added?

Reply:
Nope, since it's a global variable, at compile time it panics -- see https://hpe-aiatscale.slack.com/archives/CSG3W08JY/p1685040983218169 where Bradley went over the consequence of applying the util here